README.md

```diff
@@ -19,9 +19,9 @@ Usage documentation exists for each major version. Don't know what version you'r
 
 ## Installation
 
-Make sure you have a working Go environment.  Go version 1.11+ is supported. [See the install instructions for Go](http://golang.org/doc/install.html).
```

Comment:
I missed this line when originally making this PR, so I removed the go 1.11 reference here!